fontbakery-check-ttf.py

```diff
@@ -1976,7 +1976,11 @@ def ff_check(condition, description, err_msg, ok_msg):
                 # for the purpose of comparison we ignore any comments that
                 # may be present in the versino name entries
                 if ";" in name_version:
-                  version_without_comment, comments = name_version.split(";")
+                  version_with_comment = name_version.split(";")
+                  version_without_comment = version_with_comment[0]
+                  comments = ""
+                  for comment in [str(x) for x in version_with_comment[1:]]:
```

Comment:
Good catch! Thanks for finding this.

But the proposed fix is not really correct as it replaces ";" by spaces.
And also it is too much verbose, I think.

A good fix could be something like this:
```python
if ";" in name_version:
  version_without_comment, comments = name_version.split(";")
  comments = ";".join(comments)
```